resources/assets/app.scss

```diff
@@ -40,8 +40,8 @@ button.delete {
   }
 }
 
-.-half {
+.-third {
```

Comment:
Were we using this `.-half` class anywhere else?

We also should be careful with modifiers like this. Usually we pair a `.-x` modifier with another class (so like, `.form-item.-third`), so that it wouldn't conflict with another class using a modifier of the same name (like `.container__row.-third`).

Reply:
ah great call! I just added the `.-half` back in and created a separate element for the thirds.